example-apps/kitchen-sink-example/source/app.ts

```diff
@@ -9,19 +9,25 @@ import {bootstrap} from 'angular2/bootstrap';
 import KitchenSink from './containers/kitchen-sink';
 
 // For Batarangle Integration
-import {AppViewListener} from 'angular2/src/core/linker/view_listener';
-import {DebugElementViewListener, inspectNativeElement}
-  from 'angular2/platform/common_dom';
+// import {AppViewListener} from 'angular2/src/core/linker/view_listener';
```

Comment:
This is great, no need to modify your application to run it with Batarangle.
:+1: